CefSharp.Core/Internals/RenderClientAdapter.h

```diff
@@ -177,6 +177,13 @@ namespace CefSharp
                 _renderWebBrowser->SetCursor((IntPtr)cursor, (CefSharp::CefCursorType)type);
             };
 
+            virtual DECL bool StartDragging(CefRefPtr<CefBrowser> browser, CefRefPtr<CefDragData> drag_data,
```

Comment:
Trying to slowly bring get consistency with naming. e.g. `drag_data` becomes `dragData`